tests/unit/lib/mixins/filters.js

```diff
@@ -249,6 +249,31 @@ describe( 'mixins: filter', function() {
 
 	});
 
+	describe( '.path()', function() {
```

Comment:
Whoops -- this change is unrelated, but these tests were removed in a prior commit despite the fact that `.filter` may still be available if a client is using [this plugin](https://github.com/wp-api/rest-filter): the test coverage should be left in place since the underlying `.path()` method has not been removed.